packages/lazy-import/lib/index.ts

```diff
@@ -57,13 +57,16 @@ async function install( arg, alias ) {
  * to a temporary directory at `node_modules/.wp-lazy` relative to where the
  * current working directory.
  *
- * @param {string}                       arg       A string that you might pass
- *                                                 to `npm install`.
- * @param {Partial<WPLazyImportOptions>} [options] Optional options object.
+ * @param arg       A string that you might pass
+ *                  to `npm install`.
+ * @param [options] Optional options object.
  *
- * @return {Promise<NodeRequire>} Promise resolving to required module.
+ * @return  Promise resolving to required module.
  */
-async function lazyImport( arg, options = {} ) {
+async function lazyImport(
```

Comment:
Let us use the named export here

```suggestion
export async function lazyImport(
```